include/boost/gil/image_processing/numeric.hpp

```diff
@@ -295,6 +295,120 @@ inline void compute_hessian_entries(
     detail::convolve_2d(dy, sobel_y, ddyy);
 }
 
+namespace detail {
+enum class direction: std::size_t {
+    north = 0,
+    south = 1,
+    west = 2,
+    east = 3,
+    north_east = 4,
+    south_east = 5,
+    south_west = 6,
+    north_west = 7
+};
+
+template <typename OutputView, typename InputView>
+void compute_nabla(InputView view, const std::vector<OutputView>& nabla) {
```

Comment:
`nabla` name is cryptic, fine, but it needs a Doxygen comment on what it is.

Reply:
This is not part of public namespace (it is in detail). I'll leave a normal comment saying what nabla is, and what it is in this context.

Reply:
Okey